make/photon/prepare/utils/configs.py

```diff
@@ -458,5 +465,9 @@ def get_redis_configs(internal_redis=None, external_redis=None, with_trivy=True)
 
     if with_trivy:
         configs['trivy_redis_url'] = get_redis_url(redis['trivy_db_index'], redis)
+    
+    if 'tlsOptions' in redis and redis['tlsOptions']['rootCA'] is not None:
```

Comment:
If rootCA is not exist in the redis['tlsOptions'], it will get `KeyError: 'rootCA'`, you should use `redis['tlsOptions'].get('rootCA') is not None` to check its value.

Reply:
Sure , will update